proxy/hdrs/HTTP.cc

```diff
@@ -901,8 +902,8 @@ http_parser_parse_req(HTTPParser *parser, HdrHeap *heap, HTTPHdrImpl *hh, const
   start:
     hh->m_polarity = HTTP_TYPE_REQUEST;
 
-    // Make sure the line is not longer than 64K
-    if (scanner->get_buffered_line_size() >= UINT16_MAX) {
+    // Make sure the line is not longer than max_request_line_size
+    if (scanner->get_buffered_line_size() >= max_request_line_size) {
```

Comment:
Oh, you missed this one. It should be `>` too.

Reply:
Ah, updated it.